app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -260,6 +260,83 @@ class BookmarksTest {
         }
     }
 
+    @Test
+    fun openAllInTabsTest() {
+        val webPages = listOf(
+            TestAssetHelper.getGenericAsset(mockWebServer, 1),
+            TestAssetHelper.getGenericAsset(mockWebServer, 2),
+            TestAssetHelper.getGenericAsset(mockWebServer, 3),
+            TestAssetHelper.getGenericAsset(mockWebServer, 4),
+        )
+
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openBookmarks {
+            createFolder("root")
+            createFolder("sub", "root")
+            createFolder("empty", "root")
+        }.closeMenu {
+        }
+
+        browserScreen {
+            // out of folder and should not be opened
```

Comment:
I am not sure I really follow this comment. Could this be something that we can reword for better clarity?

Reply:
Agreed.  I'll update and move this below to where we verify that the tab is not opened.